# Define the goal
uint32 z_offset
---
# Define the result
uint32 result
---
# Define a feedback message
float32 percent_complete